src/__swaps__/screens/Swap/components/AnimatedChainImage.android.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import React, { useMemo } from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+
+const ArbitrumBadge = require('@/assets/badges/arbitrum.png');
+const BaseBadge = require('@/assets/badges/base.png');
+const BscBadge = require('@/assets/badges/bsc.png');
+const EthereumBadge = require('@/assets/badges/ethereum.png');
+const OptimismBadge = require('@/assets/badges/optimism.png');
+const PolygonBadge = require('@/assets/badges/polygon.png');
+const ZoraBadge = require('@/assets/badges/zora.png');
+const AvalancheBadge = require('@/assets/badges/avalanche.png');
+const BlastBadge = require('@/assets/badges/blast.png');
+const DegenBadge = require('@/assets/badges/degen.png');
+
+import { ChainId } from '@/__swaps__/types/chains';
+import Animated, { SharedValue, useAnimatedProps } from 'react-native-reanimated';
+import { AddressOrEth, ExtendedAnimatedAssetWithColors } from '@/__swaps__/types/assets';
+import { globalColors } from '@/design-system';
+import { customChainIdsToAssetNames } from '@/__swaps__/utils/chains';
+import { AddressZero } from '@ethersproject/constants';
+import { ETH_ADDRESS } from '@/references';
+import { PIXEL_RATIO } from '@/utils/deviceUtils';
+import { useSwapsStore } from '@/state/swaps/swapsStore';
+
+const networkBadges = {
+  [ChainId.mainnet]: EthereumBadge,
+  [ChainId.polygon]: PolygonBadge,
+  [ChainId.optimism]: OptimismBadge,
+  [ChainId.arbitrum]: ArbitrumBadge,
+  [ChainId.base]: BaseBadge,
+  [ChainId.zora]: ZoraBadge,
+  [ChainId.bsc]: BscBadge,
+  [ChainId.avalanche]: AvalancheBadge,
+  [ChainId.sepolia]: EthereumBadge,
+  [ChainId.holesky]: EthereumBadge,
+  [ChainId.optimismSepolia]: OptimismBadge,
+  [ChainId.bscTestnet]: BscBadge,
+  [ChainId.polygonAmoy]: PolygonBadge,
+  [ChainId.arbitrumSepolia]: ArbitrumBadge,
+  [ChainId.baseSepolia]: BaseBadge,
+  [ChainId.zoraSepolia]: ZoraBadge,
+  [ChainId.avalancheFuji]: AvalancheBadge,
+  [ChainId.blast]: BlastBadge,
+  [ChainId.blastSepolia]: BlastBadge,
+  [ChainId.degen]: DegenBadge,
+};
+
+export const getCustomChainIconUrlWorklet = (chainId: ChainId, address: AddressOrEth) => {
+  'worklet';
+
+  if (!chainId || !customChainIdsToAssetNames[chainId]) return '';
+  const baseUrl = 'https://raw.githubusercontent.com/rainbow-me/assets/master/blockchains/';
+
+  if (address === AddressZero || address === ETH_ADDRESS) {
+    return `${baseUrl}${customChainIdsToAssetNames[chainId]}/info/logo.png`;
+  } else {
+    return `${baseUrl}${customChainIdsToAssetNames[chainId]}/assets/${address}/logo.png`;
+  }
+};
```

Comment:
Looks like this isn't used outside of the chain badge, and it's not used in the Android version, so think this can be deleted